akka-persistence/src/test/java/akka/persistence/fsm/AbstractPersistentFSMTest.java

```diff
@@ -407,23 +407,30 @@ public String identifier() {
     // #customer-states
 
     // #customer-states-data
-    static class ShoppingCart {
+    public static class ShoppingCart {
```

Comment:
Still kinda confusing with the 'static', would be good to move this to separate files, but that's for another PR ;)